lua/lspconfig/serve_d.lua

```diff
@@ -0,0 +1,22 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.serve_d = {
+  default_config = {
+    cmd = { 'serve-d' },
+    filetypes = { 'd' },
+    root_dir = function(fname)
+      return util.root_pattern('dub.json', 'dub.sdl', '.git')(fname)
+    end,
+  },
+  docs = {
+    description = [[
+           https://github.com/Pure-D/serve-d
+
+           `Microsoft language server protocol implementation for D using workspace-d.`
```

Comment:
Are there installation instructions?

Reply:
Yes, at https://github.com/Pure-D/serve-d#installation

Reply:
Yes, I mean put the installation instructions here :)